src/Microsoft.ML.StandardLearners/Standard/MultiClass/MetaMulticlassTrainer.cs

```diff
@@ -20,7 +20,7 @@ public abstract class MetaMulticlassTrainer<TTransformer, TModel> : ITrainerEsti
         where TTransformer : ISingleFeaturePredictionTransformer<TModel>
         where TModel : class
     {
-        public abstract class OptionsBase
+        internal abstract class OptionsBase
```

Comment:
Why do we internalize an `Options` of a trainer?

Reply:
This is internal because the only class that derives from it, Options for pkpd and ova are both internal. 

The option classes are internal because there are no advanced options and all of the arguments are exposed in the argument-based constructor.


---
In reply to: [261015420](https://github.com/dotnet/machinelearning/pull/2743#discussion_r261015420) [](ancestors = 261015420)